assets/dev/js/editor/utils/jquery-html5-dnd.js

```diff
@@ -275,6 +283,17 @@
 			return true;
 		};
 
+		var disableDragFromPreviewToNewSection = function( event ) {
```

Comment:
Move the logic for 'isDroppingAllowed' into the setting of the `add-section/base.js` file.

Reply:
Done